src/notebook/notebookFeature.ts

```diff
@@ -39,8 +59,8 @@ export class JuliaNotebookFeature {
         // Add one controller per Julia minor version that we found
         for (const [kernelId, juliaVersion] of this._juliaVersions) {
             const ver = juliaVersion.getVersion()
-
-            const controller = vscode.notebooks.createNotebookController(kernelId, 'jupyter-notebook', `Julia ${ver} Kernel`)
+            const displayName = `Julia ${ver} Kernel`;
```

Comment:
Do we need the suffix `Kernel`, without that, its identical to the name in `Jupyer`

Reply:
Agreed, we can remove and use the same name as in Jupyter.